custom_components/tuya_local/devices/gorilla_fan_v2.yaml

```diff
@@ -0,0 +1,58 @@
+name: Gorilla Fan V2
```

Comment:
Preferably there should be a products section here, with product id and name (with branding), and the top level name should be generic (Ceiling fan, for example)

Reply:
I see from #1971 that the brand is Atomberg, and the product name is Gorilla.  The file naming and the name in the products section should reflect this.  I'm not sure whether the V2 is part of the product name, or it is a misunderstanding that whatever config you originally installed this as is "V1" (but there is no atomberg or gorilla config, so there is no need to use v2 here as disambiguation if it is not part of the actual product name).

Reply:
The fan in question is from Atomberg and is referred to as the Renesa fan. However, this name is not present in any of our systems. In both the Tuya Dashboard and the Smart Life App, it is listed as the Gorilla Fan V2. Therefore, I used this name for consistency.

Reply:
What about the product id? If you cannot get it from the cloud, it is also possible to read from `python -m tinytuya scan`